src/unix/lwt_unix.h

```diff
@@ -15,8 +15,10 @@
 #include <string.h>
 
 #if OCAML_VERSION < 50000
```

Comment:
```suggestion
// The following macro is for backwards compatibility.
// It is given an `lwt_` prefix to avoid name collisions for code which
// include both this file and alloc.h.
#if OCAML_VERSION < 50000
```